conf/report/filter_ui_test.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import sys
+
+def setDriver(driver=None):
+    if driver == None:
+        return webdriver.Chrome()
+    else:
+        return driver
```

Comment:
This should set global variable `driver`. Otherwise there's no way to actually set the driver from another script.